File: engine/engine-model/src/main/scala/tools/packet/ui/ShowNotes.scala
package tools.packet.ui

import client.database.data.NoteData
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowNotes(private var _notes: java.util.List[NoteData]) extends PacketInput {
  def notes: java.util.List[NoteData] = _notes

  override def opcode(): SendOpcode = SendOpcode.MEMO_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/ui/ShowNotes.java
package tools.packet.ui;

import java.util.List;

import client.database.data.NoteData;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowNotes(List<NoteData> notes) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MEMO_RESULT;
   }
}